  <launch>

    <arg name="goal_x" default="0.2"/>
    <arg name="precision_x" default="0.01"/>
    <arg name="k_l" default="1"/>

    <node pkg="go_to_goal_tut" type="go_to_goal_tut" name="go_to_goal_tut" output="screen">
    	<param name="Goal_x" value="$(arg goal_x)"/>
    	<param name="Precision_x" value="$(arg precision_x)"/>
    	<param name="K_l" value="$(arg k_l)"/>
    </node>

  </launch>
